src/components/TinyMCEEditor.jsx

```diff
@@ -92,7 +92,7 @@ export default function TinyMCEEditor(props) {
         browser_spellcheck: true,
         a11y_advanced_options: true,
         autosave_interval: '1s',
-        autosave_restore_when_empty: true,
+        autosave_restore_when_empty: false,
```

Comment:
why mark it as false when you can reset the form on cancel/submit?

Reply:
Yes, I'm resetting the form state on cancel/submit but This plugin stores the draft value in browser storage. If it is true it will get the draft value from the browser storage after the editor initialization.